ActionMenu/ActionMenu.cs

```diff
@@ -170,9 +170,10 @@ public ItemAction BuildButtonItem(string name, Action callback, bool exclusiveOp
             /// </summary>
             /// <param name="name"></param>
             /// <param name="callback"></param>
+            /// <param name="startingValue"></param>
             /// <returns></returns>
-            public ItemAction BuildToggleItem(string name, Action<bool> callback)
-                => BuildBoolItem(name, callback, "toggle", value: true, defaultValue: false);
+            public ItemAction BuildToggleItem(string name, Action<bool> callback, bool startingValue = true)
+                => BuildBoolItem(name, callback, "toggle", value: startingValue, defaultValue: false);
```

Comment:
what you're asking is defaultValue. We should add as a parameter indeed and fix #26